docs/index.md

```diff
@@ -86,6 +86,37 @@ print(Settings().model_dump())
 
     Check the [Environment variable names documentation](#environment-variable-names) for more information.
 
+## Additionnal config values
```

Comment:
I would suggest removing this section and adding doc for `_case_sensitive` to the section `### Case-sensitivity` and `_env_prefix` to the section `## Environment variable names`.

The related config keys are documented there

Reply:
Ok will do